pkgs/applications/networking/sync/unison/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation (rec {
     sha256 = "sha256-42hmdMwOYSWGiDCmhuqtpCWtvtyD2l+kA/bhHD/Qh5Y=";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper ] ++ (lib.lists.optional enableX11 copyDesktopItems);
```

Comment:
We don't need the () and lib.optional is normally used instead.

Reply:
I've amended the commit, and I'll keep it in mind for the future.